contrib/basicauth/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+nginx:
+  build: "nginx"
+  ports:
+    - "5441:5441"
+  links:
+    - registryv2:registryv2
+registryv2:
+  build: "../../"
+  ports:
+    - "5000"
```

Comment:
Yes, it should work, but the registry will also be exposed on a random port on the host *without* authentication, as can be seen in the output on line 480;

    4f4aeba1db03        basicauth_registryv2:latest   "registry cmd/regist   About a minute ago   Up About a minute   0.0.0.0:32768->5000/tcp                   basicauth_registryv2_1  

The registry will (in the above case) be accessed directly on `<docker-host>:32768`